src/main/java/net/quasardb/qdb/QdbCluster.java

```diff
@@ -200,6 +200,18 @@ public void trimAll(int timeoutMillis) {
         qdb.trim_all(session.handle(), timeoutMillis);
     }
 
+    /**
+     * Wait for all nodes of the cluster to be stabilized.
+     *
+     * @param timeoutMillis The timeout of the operation, in milliseconds
+     *
+     * @throws ClusterClosedException If QdbCluster.close() has been called.
+     */
+    public void waitForStabilization(int timeoutMillis) {
```

Comment:
`int`? You kill innocent puppies every time you don't use a strongly typed unit type... ;)